Source/Scene/Model.js

```diff
@@ -119,7 +119,7 @@ define([
 
         this.decompressedViewsToCreate = new Queue();
         this.decompressedViews = {};
-        this.decompressionInFlight = 0;
+        this.decompressionInFlight = false;
```

Comment:
I didn't look at this closely, but does this mean we can only use one web worker for decompression?